tests/e2e/specs/saved-cards/saved-cards-checkout.spec.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import { CustomerFlow, uiUnblocked } from '../../utils';
+import {
+	fillCardDetails,
+	confirmCardAuthentication,
+} from '../../utils/payments';
+
+const TIMEOUT = 100000;
+
+const cards = [
+	[ 'basic', config.get( 'cards.basic' ) ],
+	[ '3DS', config.get( 'cards.3ds' ) ],
+];
+
+describe( 'Saved cards ', () => {
+	describe.each( cards )(
+		'when using a %s card added on checkout',
+		( cardType, card ) => {
+			beforeAll( async () => {
+				// Increase default value to avoid test failing due to timeouts.
+				page.setDefaultTimeout( 30000 );
+				// running the login flow takes more than the default timeout of 5 seconds,
+				// so we need to increase it to run the login in the beforeAll hook
+				jest.setTimeout( TIMEOUT );
```

Comment:
I've noticed we have different timeout values scattered across the specs. How about moving those settings to `jest-setup.js` beforeAll/beforeEach?

Reply:
That'd be nice. Done in https://github.com/Automattic/woocommerce-payments/pull/816/commits/b909b29e6c577f4c968a0da0bfa6521ac0b248b9. Having the timeouts set up there mess the interactive tests, but as per our Slack chat, we can figure out how to have the setup run for them as well.

Reply:
@vbelolapotkov - https://github.com/Automattic/woocommerce-payments/pull/816/commits/74e8c64d967f5c5e563c74196566c28c67f56c05 seems to be enough to have the same config be run in the interactive tests. Is there any reason to not do this?

Reply:
Nice refactoring! As mentioned in our slack conversation I don't remember exact reason why we shouldn't apply custom config for interactive mode but it works fine at the moment (tested locally). So let's keep it.